hub/constants.py

```diff
@@ -7,6 +7,7 @@
 GB = 1000 * MB
 
 DEFAULT_DTYPE = "float64"
+DEFAULT_COMPRESSION = "LZ4"
```

Comment:
nit-pick can we lower-case these?

Reply:
Image compression names from meta info are uppercase, so not sure if we should lowercase them instead of making our compressors uppercase.

Reply:
all good